R/createBasicProject.R

```diff
@@ -24,6 +24,7 @@
 
   # for later resetting the project
   current_proj <- get_current_proj()
+  current_wd <- getwd()
```

Comment:
Resetting the working directory (and not only the current project) was needed when there was no `renv::scaffold()` function but I thought we could keep it "just in case".